jdi-light/src/main/java/com/epam/jdi/light/settings/TestProperties.java

```diff
@@ -0,0 +1,263 @@
+    testsProperties.put("headless",
+        new Pair<>(p -> DRIVER.capabilities.common.put("headless", p),doNothing));
+  }
+
+  private static Strategies getStrategy(String prop) {
+    String strategy = prop.trim().toLowerCase();
+    switch (strategy) {
+      case "jdi":
+        return JDI;
+      case "selenium":
+        return SELENIUM;
+      default:
+        return JDI;
+    }
+  }
+
+  private static List<Strategy> getLoggerStrategy(String strategy) {
+    if (isBlank(strategy))
+      return new ArrayList<>();
+    List<com.epam.jdi.light.logger.Strategy> strategies = new ArrayList<>();
+    try {
+      String[] split = strategy.split(";");
+      strategies = LinqUtils.map(split, s -> parseStrategy(s.trim()));
+    } catch (Exception ignore) { }
+    return strategies;
+  }
+
+  private static void setSearchStrategy(String p) {
+    p = p.toLowerCase();
+    if (p.equals("soft"))
+      p = "any, multiple";
+    if (p.equals("strict"))
+      p = "visible, single";
+    if (p.split(",").length == 2) {
+      List<String> params = asList(p.split(","));
+      if (params.contains("visible") || params.contains("displayed"))
+        onlyVisible();
+      if (params.contains("any") || params.contains("all"))
+        noValidation();
+      if (params.contains("enabled"))
+        visibleEnabled();
+      if (params.contains("inview"))
+        inView();
+      if (params.contains("single"))
+        STRICT_SEARCH = true;
+      if (params.contains("multiple"))
+        STRICT_SEARCH = false;
+    }
+  }
+
+  private static PageLoadStrategy getPageLoadStrategy(String strategy) {
+    switch (strategy.toLowerCase()) {
+      case "normal": return NORMAL;
+      case "none": return NONE;
+      case "eager": return EAGER;
+    }
+    return NORMAL;
+  }
+
+  private static ElementArea getClickType(String type) {
+    ElementArea clickType = first(getAllEnumValues(ElementArea.class),
+        t -> t.toString().equals(type));
+    return clickType != null
+        ? clickType : CENTER;
+  }
+
+  private static TextTypes getTextType(String type) {
+    TextTypes textType = first(getAllEnumValues(TextTypes.class),
+        t -> t.toString().equals(type));
+    return textType != null
+        ? textType : SMART_TEXT;
+  }
+
+  private static SetTextTypes getSetTextType(String type) {
+    SetTextTypes textType = first(getAllEnumValues(SetTextTypes.class),
+        t -> t.toString().equals(type));
+    return textType != null
+        ? textType : SET_TEXT;
+  }
+
+  private static String getRemoteUrl(String prop) {
+    switch (prop.toLowerCase().replaceAll(" ", "")) {
+      case "sauce":
+      case "saucelabs":
+        DRIVER.capabilities.common = sauceCapabilities();
+        return sauceLabs();
+      case "browserstack": return browserstack();
+      default: return seleniumLocalhost();
+    }
+  }
+
+  private static com.epam.jdi.tools.pairs.Pair<String, JFunc1<String, String>> getSmartSearchFunc(String name) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unnecessary use of fully qualified name 'com.epam.jdi.tools.pairs.Pair' due to existing import 'com.epam.jdi.tools.pairs.Pair'](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5524501)